src/amo/components/AddonMoreInfo/index.js

```diff
@@ -9,33 +9,53 @@ import Link from 'amo/components/Link';
 import { getVersionById, getVersionInfo } from 'core/reducers/versions';
 import { STATS_VIEW } from 'core/constants';
 import translate from 'core/i18n/translate';
+import { fetchCategories } from 'core/reducers/categories';
 import { hasPermission } from 'amo/reducers/users';
-import type { AddonType } from 'core/types/addons';
 import { isAddonAuthor, trimAndAddProtocolToUrl } from 'core/utils';
 import Card from 'ui/components/Card';
 import DefinitionList, { Definition } from 'ui/components/DefinitionList';
 import LoadingText from 'ui/components/LoadingText';
 import { addQueryParams } from 'core/utils/url';
+import type { CategoriesStateType } from 'amo/components/Categories';
 import type { AppState } from 'amo/store';
+import type { ErrorHandlerType } from 'core/errorHandler';
 import type { AddonVersionType, VersionInfoType } from 'core/reducers/versions';
+import type { AddonType } from 'core/types/addons';
 import type { I18nType } from 'core/types/i18n';
+import type { DispatchFunc } from 'core/types/redux';
 import type { ReactRouterLocationType } from 'core/types/router';
 
+import './styles.scss';
+
 type Props = {|
   addon: AddonType | null,
   i18n: I18nType,
 |};
 
 type InternalProps = {|
   ...Props,
+  categoriesState: $PropertyType<CategoriesStateType, 'categories'>,
```

Comment:
Rather than store all the categories from the state in a prop, I think it would simplify this component if we only stored the list of related categories, in a prop called `relatedCategories`, for example. This would mean moving the logic from this component into a function in the `categories` reducer. I will comment on this more below.